packages/frontend/app/routes/assets.create.tsx

```diff
@@ -63,6 +80,22 @@ export default function CreateAssetPage() {
                     label='Withdrawal Threshold'
                     error={response?.errors.fieldErrors.withdrawalThreshold}
                   />
+                  {tenants && (
+                    <Dropdown
+                      options={tenants.map((tenant) => ({
+                        label: tenant.node.id,
+                        value: `${tenant.node.id} ${tenant.node.publicName ? `(${tenant.node.publicName})` : ''}`
```

Comment:
Good call.... came here to suggest using the name for the select after just seeing the id in the UI, but I understand the default operator just doesnt have one.